src/serverMain/resources/userFrontend/userFrontend.css

```
@@ -413,10 +413,44 @@ a, a:visited {
   background: none;
   color: #2196F3;
   margin-top: auto;
-  margin-bottom: 16px;
   cursor: pointer;
 }
 
+.centered {
+  text-align: center;
+}
+
+#live-check-in-count  {
+  font-size: 120px;
+}
+
+footer a {
+  margin-top: 16px;
+}
+
+.blinking-dot {
+  margin-bottom: 3px;
+  height: 12px;
+  width: 12px;
+  margin-right: 8px;
+  background-color: #49b832;
+  border-radius: 50%;
+  display: inline-block;
+
+  animation: blink-animation 1s step-start 0s infinite;
+  -webkit-animation: blink-animation 1s step-start 0s infinite;
+}
+
+@keyframes blink-animation {
+  50% {
+    opacity: 0.0;
+  }
+}
+@-webkit-keyframes blink-animation {
+  50% {
+    opacity: 0.0;
+  }
+}
```

```suggestion
@keyframes blink-animation {
  50% {
    opacity: 0.2;
  }
}
@-webkit-keyframes blink-animation {
  50% {
    opacity: 0.2;
  }
}
```
Would this look better?